hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyCommitRequest.java

```diff
@@ -183,21 +196,26 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
           new CacheKey<>(dbOzoneKey),
           new CacheValue<>(Optional.of(omKeyInfo), trxnLogIndex));
 
-      omResponse.setCommitKeyResponse(CommitKeyResponse.newBuilder().build());
       omClientResponse = new OMKeyCommitResponse(omResponse.build(),
-          omKeyInfo, commitKeyRequest.getClientID());
+          omKeyInfo, dbOpenKey);
```

Comment:
We can still pass here  clientID and generate openKey. Like how we used to generate open key before.

Reply:
We are anyway computing the openKey using the clientID here. So instead of passing the clientID and computing that again in OMKeyCommitResponse, we can directly pass the openKey.

Reply:
I see that we are doing similar thing for ozone key, so that is why my question. Then we can do similar thing for ozone key also then?

Reply:
We could. But since we have to pass the omKeyInfo irrespective of whether we pass ozoneKey or not, I didnt change that. 
I am ok to have it either way. Will post an update.